lib/beaker/dsl/install_utils/foss_utils.rb

```diff
@@ -434,7 +440,12 @@ def configure_puppet_on(hosts, opts = {})
         # @api private
         def install_puppet_from_rpm_on( hosts, opts )
           block_on hosts do |host|
-            install_puppetlabs_release_repo(host)
+            if host[:type] == 'aio'
+              install_puppetlabs_release_repo(host,'pc1',opts)
+              opts[:version] = 'agent' unless opts[:version]
```

Comment:
@trevor-vaughan what is `opts[:version]` doing being assigned to `'agent'`? Where else is this used such that you want make sure this is set here?